server/src/alda/lisp/model/duration.clj

```diff
@@ -70,7 +70,7 @@
         ms    (apply + (for [{:keys [type value]} note-lengths
                              :when (= type :milliseconds)]
                          value))]
-    (when (pos? beats) (set-duration beats))
+    (set-duration beats)
```

Comment:
I didn't think about millisecond durations. This is something we should probably address in the future, but for now, we can continue to ignore them.

I think changing this line to `(when beats (set-duration beats))` will do the trick.